api/scripts/getOldPolkadotRewardEras.js

```diff
@@ -331,29 +349,39 @@ async function main() {
     polkadotAPI.reducers
   )
   console.timeEnd('parsing lunie rewards')
```

Comment:
Can we maybe already remove all this benchmarking stuff? We are not using them anymore, are we?

Reply:
I think they are useful, i.e when debugging the logs in grafana.